### ISSUE TITLE
Windows executable name as indicator

### ISSUE THREAD

**Issue by sfakiana** [MISP/MISP#2310]
Hi guys, 

Is there any watchlist for cases just like svchost.exe below? 
See recent relevant tweet:
![image](https://user-images.githubusercontent.com/1293098/27763323-966b67e4-5e79-11e7-840d-26a8543cb485.png)

I was thinking about a watchlist like the internal IP addresses where a warning is presented to the analyst.
Forgive me if this is already implemented.

Cheers,
Andreas

**Comment by adulau** [comment]
Indeed, the [MISP warning-lists](https://github.com/MISP/misp-warninglists) can be used for such case of false-postive. There are already many lists but we don't have a list for most common service name. We could add it.

Thank you
